tests/src/test/resources/sims/michael/gerritslackbot/config.yaml

```diff
@@ -1,6 +1,7 @@
 # Example configuration. This is mapped to the data class [sims.michael.gerritslackbot.model.Config]
 
 # Incoming webhook URL - required
+# See https://api.slack.com/incoming-webhooks fro details
```

Comment:
Since the comments in this file are a copy/paste from the dist version, the same comments I made above apply.